app/Livewire/Project/Service/TemplateUpdate.php

```
@@ -0,0 +1,192 @@
+    {
+        $this->editorContent = (string) $this->service->docker_compose_raw;
+    }
+
+    public function apply(): void
+    {
+        if ($this->latestCompose === null) {
+            return;
+        }
+
+        $accepted = array_map('intval', array_keys(array_filter($this->acceptedHunks)));
+        $merged = ComposeDiff::apply((string) $this->service->docker_compose_raw, $this->latestCompose, $accepted);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Services\ComposeDiff' in method 'apply'. (undefined)

(StaticAccess)